target/linux/ramips/image/mt7621.mk

```diff
@@ -193,6 +198,27 @@ define Device/dlink_dir-860l-b1
 endef
 TARGET_DEVICES += dlink_dir-860l-b1
 
+define Device/dlink_dir-878-a1
+  $(Device/ubootpad96)
+  MTK_SOC := mt7621
+  BLOCKSIZE := 64k
+  IMAGE_SIZE := 16000k
+  DEVICE_VENDOR := D-Link
+  IMAGES += factory.bin
+  DEVICE_MODEL := DIR-878
+  DEVICE_VARIANT := A1
+  DEVICE_PACKAGES := kmod-mt7615e kmod-usb3 kmod-usb-ledtrig-usbport wpad-basic \
+  		uboot-envtools
+  KERNEL_INITRAMFS := $$(KERNEL) | ubootpad96
+  IMAGE/sysupgrade.bin := append-kernel | append-rootfs |\
+			ubootpad96 |\
```

Comment:
This one could be wrapped more nicely...

Reply:
ok